doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -58,113 +58,41 @@ sdk\<service name>\<package name>\CHANGELOG.md
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
 
-### Use Cadl as Input
+### Use Cadl as Input API spec
 
-#### Create Cadl project
+#### Before you Start
   
-  You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
+  Make sure you have cadl project in [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow [Cadl Getting Start](https://github.com/microsoft/typespec/#using-node--npm) to initialize your Cadl project, and refer to [Cadl Structure Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md) to configure your Cadl project.
 
-  ***Initialize Cadl Project***
+  We will generate SDK library under SDK project folder `sdk\<servie name>\<package name>` of `azure-sdk-for-net` repo. e.g. /home/azure-sdk-for-net/sdk/anomalydetector/Azure.AI.AnomalyDetector).
 
-  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
+  Make sure that the SDK project folder exists in [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repo. If the SDK project folder does not exist, you can refer to [SDK project directory Set up](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/AzureSDKPackage_Setup.md) to create one.
 
-  Make sure `npx cadl compile .` runs correctly.
+#### Generate SDK
 
-  ***Add cadl-csharp emitter***
+We will generate SDK under the SDK project directory.
 
-  Modify package.json, add one line under dependencies:
+***configuration (optional)***
 
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
-        },
-```
-
-        Run `npm install` again to install @azure-tools/cadl-csharp.
-
-  **Notes**: @azure-tools/cadl-csharp: "0.1.8" only works with @cadl-lang/compiler: "0.37.0" and @cadl-lang/rest: "0.19.0"
-  
+You can update `cadl-location.yaml` under sdk project directory to set the cadl project.
 
-  ***Modify (or create) cadl-project.yaml, add one line under emitters:***
+You can refer to the [cadl-location.yaml](https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/Cadl-Project-Scripts.md#cadl-locationyaml) which describes the supported properties in the file.
 
-  ```diff
-  emitters:
-+  "@azure-tools/cadl-csharp": true
-  ```
+**Note**: If you want to use a local specs repo that is already present, you can specify `spec-root-dir` to provide the path of the root directory of the specs repo, e.g. /home/azure-rest-api-specs, and don't provide the `commit` and `repo`.
```

Comment:
remove this, `spec-root-dir` is an unsupported scenario.

Reply:
removed